### PR TITLE
refactor(experimental): add the rest of the content in `rpc-types/src/*` to the README

### PR DESCRIPTION
**Pull request by steveluscher**

### CODE REVIEW

**Comment by steveluscher** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1770** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1770" target="_blank">Graphite</a> 
    * **PR #1772** <a href="https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1772" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1772?utm_source=stack-comment).

File: packages/rpc-types/README.md [hunk 1]
```diff
@@ -15,3 +15,144 @@
...
+
+### `Commitment`
+
+A type that enumerates the possible commitment statuses &ndash; each a measure of the network confirmation and stake levels on a particular block. Read more about the statuses themselves, [here](https://docs.solana.com/cluster/commitments).
+
+### `LamportsUnsafeBeyond2Pow53Minus1`
+
+This type represents an integer value denominated in Lamports (ie. $1 \times 10^{-9}$ &#x25CE;).
+
+**Note**: Despite the fact that this type is represented as a `bigint` in client code and a `u64` in server code, the JSON-RPC implementation represents Lamports as an IEEE 754 floating point number. This means that you can only safely send or receive values up to $2^{52} - 1$ between the client and the RPC server. See https://github.com/solana-labs/solana/issues/30741 for more detail.
```
Comment (mcintyre94) [inline packages/rpc-types/README.md:29]: Nit: `2^53`. Sorry, I've seen that type name too many times 😅

**Review by mcintyre94 — APPROVED** [review]
Nice, I think these docs will be really helpful!

**Comment by steveluscher** [comment]
### Merge activity

* **Oct 23, 9:23 AM**: [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/1772) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**